src/Microsoft.AspNet.Authentication.Google/GoogleAuthenticationOptions.cs

```diff
@@ -28,5 +29,12 @@ public GoogleAuthenticationOptions()
         /// access_type. Set to 'offline' to request a refresh token.
         /// </summary>
         public string AccessType { get; set; }
+
+        /// <summary>
+        /// Defines whether access and refresh tokens should be stored in the
+        /// <see cref="ClaimsPrincipal"/> instance after a successful authentication.
+        /// This option is turned off by default to reduce the size of the final authentication cookie.
+        /// </summary>
+        public override bool SaveTokensAsClaims { get; set; } = false;
```

Comment:
Okay, I'll revert this change then, and merge the two comments to indicate that the default value used by the social providers is not the same as the one used by the OAuth2 base middleware.

Any remark concerning the new generic approach?